ansible/playbooks/k8s-all.yml

```diff
@@ -1,4 +1,5 @@
 - import_playbook: common.yml
 - import_playbook: master.yml
 - import_playbook: cni.yml
+- import_playbook: cri.yml
```

Comment:
The `import_playbook` list runs sequentially and therefore the CRI should be before the master playbook is executed and after the common.